Dockerfile

```diff
@@ -1,12 +1,15 @@
 ARG STELLAR_CORE_VERSION=19.3.0-1006.9ce6dc4e9.focal
 ARG HORIZON_VERSION=2.20.0-296
 ARG FRIENDBOT_VERSION=horizon-v2.20.0
+ARG SOROBAN_RPC_VERSION=horizon-v2.20.0
```

Comment:
It looks like the Dockerfile imports the rpc server from Horizon v2.20, but there is no RPC server included at that point in time: https://github.com/stellar/go/tree/horizon-v2.20.0/exp/services.

The `Dockerfile` is really only supposed to contain released versions of software. It might be pertinent to comment the RPC lines in the Dockerfile (leave them there, we will need them, so maybe just comment them and leave a TODO message).

People will be able to use the `:testing` image to get it.

Once the RPC server has been released, we should use its version here, not Horizons. (Friendbot uses Horizon's version because it is really a sub-service of Horizon.)

Reply:
yes, soroban-rpc version had override in Dockerfile.yml to be set as `latest` to get master instead for now, but, I will follow this suggestion and TODO out soroban-rpc from Dockerfile and Dockerfile.yml, I do have `SOROBAN_RPC_VERSION=latest` still for Dockerfile.testing.yml.

Reply:
ok, commented out refs to soroban-rpc in Dockerfile  - https://github.com/stellar/quickstart/pull/367/commits/8019a99efc33bdb864f8bf78b9d8d8a94d8af74e